### PR TITLE
Connecting PG database

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

File: knex/knexfile.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = {
   production: {
     client,
     connection: process.env.DATABASE_URL,
+    ssl: { rejectUnauthorized: false },
```
Comment (geramirez) [inline knex/knexfile.js:21]: had to add this because we are using the free tier that doesn't support ssl... I think

File: server/index.js [hunk 2]
```diff
@@ -12,64 +12,77 @@ const app = express()
 app.use(express.json());
 const PORT = process.env.PORT || 5000
 
-const db = new loki('local.db');
 
 const twentyFourHoursAgo = () => (new Date) - 60 * 60 * 1000 * 24
 
-const getParkLocation = (key) => (
-  {
-    'bartholomew': { latitude: 30.305177, longitude: -97.697469 },
```
Comment (geramirez) [inline server/index.js:21]: thanks for updating the configuration!

File: server/index.js [hunk 3]
```diff
@@ -12,64 +12,77 @@ const app = express()
...
-    const workingVotes = this.votes.count({ status: true, timestamp: { $gte: twentyFourHoursAgo() }, parkKey })
-    const notWorkingVotes = this.votes.count({ status: false, timestamp: { $gte: twentyFourHoursAgo() }, parkKey })
-
-    if (workingVotes === 0 && notWorkingVotes === 0) {
-      const orderedVotes = this.votes.chain().find({ parkKey }).simplesort('timestamp', { desc: true }).data()
-      if (orderedVotes.length === 0 || orderedVotes[0].status === false) {
-        return { status: "unknown", workingVotes, notWorkingVotes }
+  async getStatus(parkName) {
+    const park_id = await this.getParkId(parkName)
+    const lastVoteRecords = await knex
```
Comment (geramirez) [inline server/index.js:57]: there are probably better ways to do these queries, but this works for now. thoughts?

**Comment by geramirez** [comment]
@kiranjthomas thoughts?

File: server/index.js (hunk 3 above)
Comment (kiranjthomas) [reply server/index.js:57]: I'm actually new to the knex query builder. This looks good for now though!

**Review by kiranjthomas — COMMENTED** [review]
Lgtm

**Pull request merged**